### PR TITLE
Clean up redirects for 'docs/user-guide/...' entries

### PR DESCRIPTION
**Pull request by tengqm**
This is the second PR for cleaning up old redirection entries that are more than 4 years old. The `docs/user-guide` directory was removed in June, 2018. It should be okay to remove these entries since those old bookmarks are no longer useful anyway.

### CODE REVIEW

**Comment by natalisucks** [comment]
Thanks for the cleanup PRs, Qiming – very appreciated.
/lgtm

File: content/en/docs/reference/glossary/kubectl.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 title: Kubectl
 id: kubectl
 date: 2018-04-12
-full_link: /docs/user-guide/kubectl-overview/
+full_link: content/en/docs/reference//kubectl/kubectl/
```
Comment (kbhawkey) [inline content/en/docs/reference/glossary/kubectl.md:5]: Is this updated link correct?

**Comment by kbhawkey** [comment]
Should be able to remove the hold
/hold cancel

File: content/en/docs/reference/glossary/kubectl.md (hunk 1 above)
Comment (tengqm) [reply content/en/docs/reference/glossary/kubectl.md:5]: No. Let me fix that.

**Pull request merged**